astropy/config/paths.py

```diff
@@ -71,21 +71,31 @@ def _find_home():
             homedir = os.environ['HOME']
         else:
             raise OSError('Could not find a home directory to search for '
-                          'astropy config dir - are you on an unspported '
+                          'astropy config dir - are you on an unsupported '
                           'platform?')
     return homedir
 
 
-def get_config_dir(create=True):
+def get_config_dir(rootname, create=True):
```

Comment:
This is a public API function so this is a breaking API change.  I think `rootname` needs to default to `astropy`, and be the second argument.  I realize this isn't used that much, but I think it *is* used by some affiliated packages that use tricks to do something else.